src/constants/permissions.ts

```diff
@@ -0,0 +1,53 @@
+export const Permissions = {
+  Reports: {
+    Score: {
+      ALL: 'reports.score.*',
+      READ: 'reports.score.read',
+    },
+    Progress: {
+      ALL: 'reports.progress.*',
+      READ: 'reports.progress.read',
+    },
+    Student: {
+      ALL: 'reports.student.*',
+      READ: 'reports.student.read',
+    },
+  },
+  Organizations: {
+    ALL: 'organizations.*',
+    LIST: 'organizations.list',
+    CREATE: 'organizations.create',
+  },
+  Administrations: {
+    ALL: 'administrations.*',
+    LIST: 'administrations.list',
+    CREATE: 'administrations.create',
+    UPDATE: 'administrations.update',
+  },
+  Administrators: {
+    ALL: 'administrator.*',
+    READ: 'administrator.READ',
+    CREATE: 'administrator.create',
+    UPDATE: 'administrator.update',
+    Credentials: {
+      UPDATE: 'administrator.credentials.update',
+    },
+  },
+  Profile: {
+    ALL: 'profile.*',
+    READ: 'profile.READ',
+  },
+  Users: {
+    ALL: 'users.*',
+    LIST: 'users.list',
+    CREATE: 'users.create',
+    UPDATE: 'users.update',
+    Credentials: {
+      UPDATE: 'users.credentials.update',
+    },
+  },
+  Tasks: {
+    ALL: 'tasks.*',
+    MANAGE: 'tasks.manage',
```

Comment:
`MANAGE` is confusing here as it would have the same significance than `ALL`. We should stick to CRUD-like actions as discussed before 

```suggestion
    ALL: 'tasks.*',
    MANAGE: 'tasks.manage',
```

Reply:
I used manage here as I did not feel it was necessary to split the permissions for UPDATE and CREATE. However I can split those if you think that's appropriate.